css/aboutme.scss

```diff
@@ -0,0 +1,69 @@
+body {
+}
+
+#team {
```

Comment:
The indents in this file are off, can you double check that they're 2 space indents?